src/admin/components/UserResumeTable/index.tsx

```diff
@@ -79,11 +110,28 @@ const UserResumeTable: React.FC<UserResumeTableProps> = (props) => {
     {
       title: 'Planned Graduation',
       dataIndex: 'user.graduationYear',
+
+      filterMultiple: true,
+      filters: yearFilters, // [{ text: "2016", value: "2016" }],
+
+      onFilterDropdownVisibleChange: (visible) => {
+        if (visible) {
+          setTimeout(() => updateYearFilters());
+        }
+      },
+
+      // Value we are filtering by, record = content of current row
+      onFilter: (value: string, record: UserResume) => `${record.user.graduationYear}` === value,
     },
     {
       title: 'Date Uploaded',
       dataIndex: 'lastUpdated',
       render: (date: string) => <span>{reformatDate(date)}</span>,
+
+      onFilter: (value, record) => record.name.indexOf(value) === 0,
+      sorter: (a, b) => {
+        return a.lastUpdated.localeCompare(b.lastUpdated);
```

Comment:
We can inline this function's return value

Reply:
We should also use proper variable names that are representative of the data like (user1, user2) or resumeEntry, etc for whatever this is rather than `a, b`